code/solutions/package-and-deploy/scripts/invoke-lambda-function.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# From code/package-and-deploy directory run ./scripts/invoke-lambda-function.sh
+
+LAMBDA_FUNCTION_NAME="cfn-workshop-python-function"
+
+PAYLOAD='{"time_zone":"Europe/London"}'
+
+aws lambda invoke \
+  --function-name $LAMBDA_FUNCTION_NAME \
+  --payload $PAYLOAD \
+  response.json
+
+echo "$(<response.json)"
```

Comment:
cat?

Reply:
or use stdout as the file for the preceding command?

Reply:
Updated